### DISCUSSION TITLE
itwinviewer proxy app

### DISCUSSION THREAD

**Discussion by Rahul-0108** [iTwin/itwinjs-core#5358] [Q&A] [answered]
I tried to use the itwinviewer proxy concept in my app, everything worked , except this network call fails
![image](https://user-images.githubusercontent.com/53996840/230348725-83842e49-c43d-4d4c-a084-980a4b0b32c7.png)

Any soln to this ??

**Comment by pmconne** [comment, marked as answer]
This is a request to retrieve a cached tile from Azure blob storage. The 404 indicates the tile has not been cached; in response, your app will send a request to the iModel backend to generate+cache the tile. All is behaving as expected.

Unfortunately there is no way to prevent such "errors" from showing up in the console output.
